keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -226,6 +233,28 @@ def augment_bounding_boxes(
     def augment_label(self, label, transformation, **kwargs):
         return label
 
+    def augment_segmentation_mask(self, segmentation_mask, transformation, **kwargs):
+        # If segmentation_classes is specified, we have a dense segmentation mask.
+        # We therefore one-hot encode before rotation to avoid bad interpolation
+        # during the rotation transformation. Later, we make the mask sparse
+        # again using tf.argmax.
+        if self.segmentation_classes:
```

Comment:
I think we should do a check for the last column of the input mask, and raise error if it is 1. This will prevent user from accidentally miss the mask class.

Reply:
Done -- and added a test to verify checking.